tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_init_deploy.tpl

```diff
@@ -7,13 +7,19 @@
       source /mnt/secrets/static/* &&
       az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID &&
       az account set -s $AZURE_SUBSCRIPTION_ID &&
-      az deployment sub create \
-          -n {{ .Release.Name }} \
-          -l westus2 \
+      groupName='{{ lower .Scenario }}-{{ .Release.Name }}-{{ .Release.Revision }}'
+      az group create -l westus2 -g $groupName &&
+      group=$(az group show -g $groupName -o tsv --query "id") &&
+      az tag create --resource-id $group --tags DeleteAfter="$(date -d '+192:00:00' -Iseconds -u)" &&
```

Comment:
As this is growing in complexity, I plan to converge with our common deployment scripts for live tests: https://github.com/Azure/azure-sdk/issues/3211